build.sbt

```
@@ -82,6 +82,7 @@ lazy val commonSettings = Seq(
   organizationName     := "Fulcrum Genomics LLC",
   organizationHomepage := Some(url("http://www.fulcrumgenomics.com")),
   homepage             := Some(url("http://github.com/fulcrumgenomics/sopt")),
+  scmInfo              := Some(ScmInfo(url("http://github.com/fulcrumgenomics/sopt"), "scm:git@github.com:fulcrumgenomics/sopt.git")),
```

Are we allowed to upgrade to `https`?
```suggestion
  scmInfo              := Some(ScmInfo(url("https://github.com/fulcrumgenomics/sopt"), "scm:git@github.com:fulcrumgenomics/sopt.git")),
```

Would be nice to have URLs above be https too.